### PR TITLE
Update docs/samples/pipelines/

### PR DESCRIPTION
**Pull request by georgetree**
**What this PR does / why we need it**:
Because KFserving was updated to KServe, the model-app-ui does not display model with using current sample code.

### CODE REVIEW

**Comment by yuzisun** [comment]
@markwinter I think you contributed the the kserve pipeline component, can you help review this?

**Review by markwinter — APPROVED** [review]
LGTM - just need to fix the lint issues
https://github.com/kserve/kserve/runs/5753506186?check_suite_focus=true

@georgetree Thanks for updating!

File: docs/samples/pipelines/sample-tf-pipeline.py [hunk 1]
```diff
@@ -15,26 +15,27 @@
...
-def kfservingPipeline(
+def kservePipeline(
         action='apply',
         model_name='tensorflow-sample',
         model_uri='gs://kfserving-samples/models/tensorflow/flowers',
         namespace='anonymous',
         framework='tensorflow'):
-    kfserving_op(action=action,
+    kserve_op(action=action,
                  model_name=model_name,
```
Comment (markwinter) [inline docs/samples/pipelines/sample-tf-pipeline.py]: Lint error here: align the parameters

File: docs/samples/pipelines/sample-custom-model.py [hunk 2]
```diff
@@ -15,25 +15,27 @@
...
-def kfservingPipeline(
+def kservePipeline(
         action='apply',
         model_name='max-image-segmenter',
         namespace='anonymous',
         custom_model_spec='{"name": "image-segmenter", "image": "codait/max-image-segmenter:latest", "port": "5000"}'
 ):
-    kfserving_op(action=action,
+    kserve_op(action=action,
                  model_name=model_name,
```
Comment (markwinter) [inline docs/samples/pipelines/sample-custom-model.py]: Lint error here: align the parameters

File: docs/samples/pipelines/sample-tf-pipeline.py (hunk 1 above)
Comment (georgetree) [reply docs/samples/pipelines/sample-tf-pipeline.py]: Thanks, I have fixed this error.

File: docs/samples/pipelines/sample-custom-model.py (hunk 2 above)
Comment (georgetree) [reply docs/samples/pipelines/sample-custom-model.py]: Thanks, I have fixed this error.

File: docs/samples/pipelines/sample-tf-pipeline.py [hunk 3]
```diff
@@ -15,26 +15,28 @@
 import kfp.dsl as dsl
 from kfp import components
 
-kfserving_op = components.load_component_from_url('https://raw.githubusercontent.com/kubeflow/pipelines/'
-                                                  'master/components/kubeflow/kfserving/component.yaml')
+# kfserving_op = components.load_component_from_url('https://raw.githubusercontent.com/kubeflow/pipelines/'
+#                                                  'master/components/kubeflow/kfserving/component.yaml')
+kserve_op = components.load_component_from_url('https://raw.githubusercontent.com/kubeflow/pipelines/'
+                                               'master/components/kserve/component.yaml')
```
Comment (markwinter) [inline docs/samples/pipelines/sample-tf-pipeline.py:21]: Lint issue: need to align this

File: docs/samples/pipelines/sample-custom-model.py [hunk 4]
```diff
@@ -15,25 +15,27 @@
 import kfp.dsl as dsl
 from kfp import components
 
-kfserving_op = components.load_component_from_url('https://raw.githubusercontent.com/kubeflow/pipelines/master/'
-                                                  'components/kubeflow/kfserving/component.yaml')
+# kfserving_op = components.load_component_from_url('https://raw.githubusercontent.com/kubeflow/pipelines/master/'
+#                                                   'components/kubeflow/kfserving/component.yaml')
+kserve_op = components.load_component_from_url('https://raw.githubusercontent.com/kubeflow/pipelines/'
+                                               'master/components/kserve/component.yaml')
```
Comment (markwinter) [inline docs/samples/pipelines/sample-custom-model.py:21]: Lint issue: need to align this

**Comment by markwinter** [comment]
@yuzisun 
I think because of force-push you might need to re-approve the tests?

**Comment by yuzisun** [comment]
Thanks @georgetree and @markwinter !

/lgtm
/approve

**Pull request merged**